cmd/minishift/cmd/config/validations.go

```diff
@@ -41,7 +41,7 @@ func IsValidDriver(string, driver string) error {
 }
 
 func RequiresRestartMsg(name string, value string) error {
-	api := libmachine.NewClient(constants.Minipath, constants.MakeMiniPath("certs"))
+	api := libmachine.NewClient(constants.GetMinipath(), constants.MakeMiniPath("certs"))
```

Comment:
Note, constants.Minipath is a global variable. Provided you set it to the right value somewhere during startup, there is no difference to calling constants.Minipath and your constants.GetMinipath(). In fact, provided you make use of the minipath multiple times (as we do), the former is even better, since it evaluates the directory only once, instead of running the same code to determine the directory over and over again.